docs/source/conf.py

```diff
@@ -346,17 +347,22 @@ def inject_weight_metadata(app, what, name, obj, options, lines):
             lines.append("")
 
 
-def generate_classification_table():
+def generate_weights_table(module, table_name, metrics_names):
 
-    weight_enums = [getattr(M, name) for name in dir(M) if name.endswith("_Weights")]
+    weight_enums = [getattr(module, name) for name in dir(module) if name.endswith("_Weights")]
     weights = [w for weight_enum in weight_enums for w in weight_enum]
 
-    column_names = ("**Weight**", "**Acc@1**", "**Acc@5**", "**Params**", "**Recipe**")
+    def clean_name(name):
```

Comment:
Nit: Might be cleaner if you pass `metrics_names` as `Tuple[str, str]` where one part of the tuple is the key and the second the doc-friendly name of it.